sections/DACs_references.md

```diff
@@ -2,3 +2,18 @@
 
 ```{bibliography}
 ```
+
```

Comment:
Maybe add something like

## Additional key literature

Reply:
true, I'll do that and update the table with proper way of citing

Reply:
ok so we can close this PR?